tools/substitute_tar_deps.bzl

```diff
@@ -33,7 +33,7 @@ def substitute_tar_deps(name, src, pkg_deps, out):
                 pkg_dep,
             ],
             cmd = """
-                TAR=$$(dirname $$(readlink $(execpath {pkg_dep})))/npm_package_archive.tar.gz
+                TAR=$$(dirname $$(cd $$(dirname $(execpath {pkg_dep})) && pwd))/npm_package_archive.tar.gz
```

Comment:
@gregmagolan This readlink trick to get the absolute tar path worked locally but was failing on circleci. It seems that on the ci environment the action isn't run in a sandbox, so there is no symlink from the copy of package.json in the sandbox folder to the exec root. Using this trick instead (cd'ing in to the directory and calling pwd) works on circleci but not locally because locally it will reference the sandbox directory which gets deleted.

I'm a bit confused on what to do here. Should the ci environment be executing actions in a sandbox?